pkg/gds/service_test.go

```diff
@@ -255,10 +345,13 @@ func (s *gdsTestSuite) CompareFixture(namespace, key string, obj interface{}, re
 	switch namespace {
 	case vasps:
 		var a *pb.VASP
+		var data pb.VASP
 		for _, f := range s.fixtures[namespace] {
 			ref := f.(*pb.VASP)
 			if ref.Id == key {
-				a = ref
+				// Avoid modifying the object in the fixtures map
+				data = *ref
+				a = &data
```

Comment:
Fixes a bug I ran into with one of the tests calling `CompareFixtures` which inadvertently altered one of the VASP fixtures and caused a subsequent test to fail.